goldendict: use ffmpeg instead of ffmpeg_3

sikmir opened the pull request:
###### Motivation for this change
* use ffmpeg instead of ffmpeg_3 (#120705)
* [changelog](https://github.com/goldendict/goldendict/compare/261e45a5d79f9df2fbc050292410bed0f4ef3132...b2e673961d28ca5eb920a909091252d3321f09d6)

###### Things done
- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

sikmir committed:
goldendict: 2020-12-09 → 2021-03-09, use ffmpeg instead of ffmpeg_3

r-rmcgibbo commented:
Result of `nixpkgs-review pr 123527` at 55c19deb run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built successfully:</summary>
  <ul>
    <li>goldendict</li>
    <li>tests.trivial</li>
  </ul>
</details>
<details>
  <summary>2 suggestions:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-patch-comment.md">missing-patch-comment</a>

Consider adding a comment explaining the purpose of this patch on the line preceeding.
Near pkgs/applications/misc/goldendict/default.nix:23:5:
```
   |
23 |     ./0001-dont-check-for-updates.patch
   |     ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-patch-comment.md">missing-patch-comment</a>

Consider adding a comment explaining the purpose of this patch on the line preceeding.
Near pkgs/applications/misc/goldendict/default.nix:25:5:
```
   |
25 |     ./0001-dont-use-maclibs.patch
   |     ^
```
</li>
  </ul>
</details>

contrun reviewed (approved):
Thanks. Goldendict built by `nixpkgs-review pr 123527` works on my machine.

bjornfor reviewed (changes requested):
In order of preference:
1. Split out the version bump in a separate commit.
2. Say something about the version bump in the commit.

sikmir commented:
> In order of preference:
> 
>     1. Split out the version bump in a separate commit.
> 
>     2. Say something about the version bump in the commit.

Updated commit message.
